configuration/scripts/parse_namelist_from_env.sh

```diff
@@ -5,16 +5,13 @@ if [[ "$#" -ne 1 ]]; then
   exit -1
 fi
 
+scriptname=`basename "$0"`
 filename=$1
 
 #echo "$0 $1" 
-echo "running parse_namelist_from_env.sh"
+echo "running $scriptname"
 
-sed -i.sedbak -e 's|ICE_SANDBOX|'"${ICE_SANDBOX}"'|g' $filename
-sed -i.sedbak -e 's|ICE_MACHINE_INPUTDATA|'"${ICE_MACHINE_INPUTDATA}"'|g' $filename
-
-if [[ -e "${filename}.sedbak" ]]; then
-  rm ${filename}.sedbak
-fi
+sed -i -e 's|ICE_SANDBOX|'"${ICE_SANDBOX}"'|g' $filename
+sed -i -e 's|ICE_MACHINE_INPUTDATA|'"${ICE_MACHINE_INPUTDATA}"'|g' $filename
```

Comment:
this `.sedbak` extension to the `-i` argument  is needed on macOS to avoid warnings, I'd prefer we keep them (that's why they were added).

Reply:
OK, I'll test on Mac.  The sedbak slows things down a bit which is why I removed it after timing the scripts.  I was trying to speed things up.  If it breaks Mac, then we need to find a work around or use sedbak.

Reply:
I played around with sed on cheyenne as well as three flavors of sed on two Macs.  gnu-sed works fine on the Macs if it's installed, which it is on my machines.  The default sed is different.  I tried various things, again, and have decided to just revert the .sedbak implementation.  There are other things we could do, but none are ideal.